firebase-database/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 19.7.1
```

Comment:
I just realized that we can probably drop this since we never released startAfter(). I can do this manually when we work on the public release notes if you don't want to do another CI run.